codegen/aws/core/src/main/java/software/amazon/smithy/python/aws/codegen/AwsUserAgentIntegration.java

```diff
@@ -28,7 +28,7 @@ def aws_user_agent_plugin(config: $1T):
                         ua_suffix=config.user_agent_extra,
                         ua_app_id=config.sdk_ua_app_id,
                         sdk_version=$3T,
-                        service_id='$4L'
+                        service_id=\"$4L\",
```

Comment:
Just curious, why do these need to be escaped? Can't we use just `"` since this is part of a text block/multi-line string?

Reply:
Yep, good call. These don't need the escapes, this was just find-and-replace from the other changes.